cylc/flow/scripts/config.py

```diff
@@ -102,6 +103,25 @@ def get_option_parser():
             "overrides any settings it shares with those higher up."),
         action="store_true", default=False, dest="print_hierarchy")
 
+    platform_listing_options_group = parser.add_option_group(
+        'Platform printing options:')
```

Comment:
Didn't know about this method. Turns out it adds a colon for you
```suggestion
        'Platform printing options')
```